kubetest/kind/kind.go

```diff
@@ -239,23 +240,27 @@ func (d *Deployer) Build() error {
 	log.Println("kind.go:Build()")
 	// Adapt the build type if needed.
 	var buildType string
+	var buildNodeImage string
 	switch d.buildType {
 	case "":
 		// The default option is to use a pre-build image.
 		log.Println("Skipping the kind node image build.")
-		d.kindNodeImage = ""
 		return nil
 	case "quick":
 		// This is the default build type in kind.
 		buildType = "docker"
+		buildNodeImage = kindNodeImageLatest
 	default:
 		// Other types and 'bazel' are handled transparently here.
 		buildType = d.buildType
+		buildNodeImage = kindNodeImageLatest
 	}
 
 	args := []string{"build", "node-image", "--type=" + buildType, flagLogLevel, "--kube-root=" + d.importPathK8s}
-	if d.kindNodeImage != "" {
-		args = append(args, "--image="+d.kindNodeImage)
+	if buildNodeImage != "" {
+		args = append(args, "--image="+buildNodeImage)
+		// override user-specified node image
+		d.kindNodeImage = buildNodeImage
```

Comment:
I override the node image with `kindNodeImageLatest`, because I think it will confuse the user to start a cluster with `kindest/node:v1.15.0` but it might not be `kindest/node:v1.15.0`.

Reply:
i'm not sure what to do here. long term i'd like to reccomend not using kubetest for using kind.